src/components/Carousel/Carousel.tsx

```diff
@@ -0,0 +1,87 @@
+import React, { useState, useCallback } from 'react';
+
+import MobileStepperMaterial from '@mui/material/MobileStepper';
+import classNames from 'classnames/bind';
+
+import { Button } from '@components/Button';
+import { SvgIcon } from '@components/SvgIcon';
+import { SwipeableViews } from '@components/SwipeableViews';
+import { Typography } from '@components/Typography';
+
+import styles from './Carousel.module.css';
+
+const cn = classNames.bind(styles);
+
+type CarouselContent = { id: string; imagePath: string }[];
+
+export type CarouselProps = {
+  carouselContent: CarouselContent;
+  title: string;
+  className?: string;
+  imgClassName?: string;
+  stepperClassName?: string;
+};
+
+export function Carousel({ carouselContent, title, imgClassName, className, stepperClassName }: CarouselProps) {
```

Comment:
And this looks like a gallery for images, and it is ok. But we won't be able to use it in the news module, because I think we have to change not only pictures but the whole news there. With  text and action  for moving somewhere where you will be able to write more